code_challenges/code_wars/popping_balloons.py

```diff
@@ -0,0 +1,94 @@
+class Balloons:
+    def __init__(self, amount, balloons):
+        self.balloons = balloons
+        self.amount = amount
+        self.balloons_amount = {}
+        self.popped_ballons = []
+
+    
+    def track_balloons(self):
+        for balloon in self.balloons:
+            self.balloons_amount[balloon] = self.balloons_amount.get(balloon, 0) + 1
+        return self.balloons_amount
+
+    def find_top_balloons(self):
+        top_amount = 0
+        second_amount = 0
+        top_balloon = None
+        second_balloon = None
+        
+        for balloon in self.balloons_amount:
+            if self.balloons_amount[balloon] > top_amount:
+                top_amount = self.balloons_amount[balloon]
+                top_balloon = balloon
```

Comment:
If it's greater than the top amount, then you would have a new top amount and a new second in this same if block.

Reply:
i think i have fixed this but it still fails